docs/api.md

```diff
@@ -142,23 +142,23 @@ player.playlist.contains({
 // true
 ```
 
-#### `player.playlist.indexOf(String|Object|Array value) -> Number`
+#### `player.playlist.indexOf(String|Object|Array value) -> Number|Array`
```

Comment:
Making this return an array makes this API be inconsistent with every other indexOf implementation. It should just keep returning the first index that is found.

Reply:
Thinking about it some more, this is actually a breaking change because libs that use this expect a number but if they start getting an array, their code will break. I don't think this is necessary to make things work properly for returning the correct indexOf if there are two items with the same source. Just looking for the correct id should already handle it.

Reply:
Yeah, I don't think this is going to work. It should probably return the first index it finds - this is how `indexOf` behaves for arrays.

It may make sense to add an `indexesOf` method that always returns an array.